lib/test.rb

```diff
@@ -354,7 +354,7 @@ def install_mercurial_if_needed(deps, reqs)
       end
     end
 
-    def setup_formulae_deps_instances(formula, formula_name)
+    def setup_formulae_deps_instances(formula, formula_name, still_to_test:)
```

Comment:
Rather than passing these in I'd be tempted to use a `@formula_that_have_been_built` array or something which is shared between methods.